src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -25,6 +25,9 @@ interface ConversationRouteProps {
 
 const ConstrainedHeightFlex = styled(Flex)`
   height: calc(100vh - 145px);
+  ${media.xs`
+    height: calc(100vh - 55px);
+  `}
```

Comment:
There was too much white space on the bottom of the messages page on mobile so had to adjust this.